nepal_compliance/cbms_api.py

```
@@ -198,11 +277,36 @@ def post_sales_invoice_or_return_to_cbms(doc_name: Any, method: Optional[str] =
-        return {"message": _("Request processed successfully"), "status": "success"}
+        return {"message": _("Request processed successfully"), "status": "queued"}
+
+    except Exception as e:
+        frappe.log_error(
+            message=str(e),
+            title="CBMS API Error"
+        )
+        return {"message": _("An error occurred while processing the request: {0}").format(str(e)), "status": "error"}
+
+@frappe.whitelist()
+def post_sales_invoice_status(doc_name: Any, method: Optional[str] = None) -> dict:
```

Ruff (0.15.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `doc_name`

(ANN401)